src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -391,18 +409,26 @@ impl<'b> Resolver<'b> {
         }
 
         let name = xcdef.name;
-        let vis = if parent.is_trait() { ty::Visibility::Public } else { xcdef.vis };
+        let vis = if parent.is_trait() {
+            ty::Visibility::Public
+        } else {
+            xcdef.vis
+        };
 
         match def {
-            Def::Mod(_) | Def::ForeignMod(_) | Def::Enum(..) => {
+            Def::Mod(_) |
+            Def::ForeignMod(_) |
+            Def::Enum(..) => {
                 debug!("(building reduced graph for external crate) building module {} {:?}",
-                       name, vis);
+                       name,
+                       vis);
```

Comment:
Agreed, I think it far more readable and aesthetically pleasing to format `debug!`s with long format strings as
```rust
debug!("x: {}, y: {}, z: {}, ... (long line) ...",
       x, y, z);
```
than
```rust
debug!("x: {}, y: {}, x: {}, ... (long line) ...",
       x,
       y,
       z);
```